it/test/controllers/manageAgent/AgentJourney2026FlagEnabled/AreYouSureISpec.scala

```diff
@@ -0,0 +1,518 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.manageAgent.AgentJourney2026FlagEnabled
+
+import base.{HtmlComponentHelpers, ISpecBase}
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import models.propertyrepresentation.AgentAppointmentChangesResponse
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.http.HeaderNames
+import play.api.http.Status.OK
+import play.api.libs.json.Json
+import play.api.libs.ws.WSResponse
+import play.api.test.Helpers._
+import utils.ListYearsHelpers
+
+class AreYouSureISpec extends ISpecBase with HtmlComponentHelpers with ListYearsHelpers {
+
+  override lazy val extraConfig: Map[String, String] = Map("feature-switch.AgentJourney2026" -> "true")
```

Comment:
Same as earlier, the flag is wrong, it should be feature-switch.agentJourney2026Enabled